scipy/interpolate/src/fitpack.pyf

```diff
@@ -110,34 +110,36 @@ static F_INT calc_regrid_lwrk(F_INT mx, F_INT my, F_INT kx, F_INT ky,
 
      !!!!!!!!!! Univariate spline !!!!!!!!!!!
 
-     subroutine splev(t,n,c,k,x,y,m,e,ier)
-       ! y = splev(t,c,k,x,[e])
+     subroutine splev(t,n,c,nc,k,x,y,m,e,ier)
+       ! y,ier = splev(t,c,k,x,[e])
        threadsafe
        real*8 dimension(n),intent(in) :: t
        integer intent(hide),depend(t) :: n=len(t)
-       real*8 dimension(n),depend(n,k),check(len(c)==n),intent(in) :: c
+       real*8 dimension(nc), intent(in), depend(n,k,t),check(len(c)>=n-k-1) :: c
+       integer intent(hide), depend(c) :: nc=len(c)
        integer :: k
        real*8 dimension(m),intent(in) :: x
        real*8 dimension(m),depend(m),intent(out) :: y
        integer intent(hide),depend(x) :: m=len(x)
-       integer check(0<=e && e<=2) :: e=0
-       integer intent(hide) :: ier
+       integer check(0<=e && e<=3) :: e=0
```

Comment:
I believe this is a bug. This `e` should be 0, 1, 2, 3.
https://github.com/scipy/scipy/blob/9562a4a1fd1005eefb7221d1675ef6dcedcf07b8/scipy/interpolate/_fitpack_impl.py#L540-L547